src/file-picker/src/FilePicker.js

```diff
@@ -49,7 +49,12 @@ export default class FilePicker extends PureComponent {
     /**
      * Function called when onChange is fired
      */
-    onChange: PropTypes.func
+    onChange: PropTypes.func,
+
+    /**
+     * Function called when onBlur is fired
+     */
+    onBlur: PropTypes.func
```

Comment:
If we don't add onBlur handler here, there is no easy way to attach [redux-form field properties](https://redux-form.com/6.6.3/docs/api/field.md/) to the filepicker